lab-patrick/lib/index.js

```diff
@@ -0,0 +1,6 @@
+'use strict';
```

Comment:
No points docked for this, but just make sure your ``index.js`` module is in the root of your directory at the same level as ``package.json`` and ``.gitignore.``